### PR TITLE
compile a flutter tester with SUPPORT_FRACTIONAL_TRANSLATION and raster cache disabled

### PR DESCRIPTION
**Pull request by jonahwilliams**
By turning off pixel snapping in the tester, we believe that we can stabilize a significant number of google3 goldens that get invalidated when we change how layers composite.

### CODE REVIEW

File: shell/testing/support.h [hunk 1]
```diff
@@ -0,0 +1,5 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#define SUPPORT_FRACTIONAL_TRANSLATION 1;
```
Comment (jonahwilliams) [inline shell/testing/support.h:5]: I think this is how C++ works 😄

Comment (jonahwilliams) [reply shell/testing/support.h:5]: maybe I actually need to include it in the other main. Then ... how do I do that conditionally via GN. I need to learn gn

File: shell/testing/BUILD.gn [hunk 2]
```diff
@@ -36,3 +36,40 @@ executable("testing") {
     "//third_party/skia",
   ]
 }
+
+# A second tester binary with SUPPORT_FRACTIONAL_TRANSLATION enabled.
+executable("testing_v2") {
```
Comment (dnfield) [inline shell/testing/BUILD.gn:41]: please name this something clearer. Something like

`testing_fractional_translation`.

**Review by dnfield — APPROVED** [review]
LGTM. 

Consider picking a more descriptive name.

File: shell/testing/BUILD.gn (hunk 2 above)
Comment (jonahwilliams) [reply shell/testing/BUILD.gn]: Done

File: shell/testing/BUILD.gn [hunk 3]
```diff
@@ -36,3 +36,40 @@ executable("testing") {
     "//third_party/skia",
   ]
 }
+
+# A second tester binary with SUPPORT_FRACTIONAL_TRANSLATION enabled.
+executable("testing_fractional_translation") {
+  output_name = "flutter_tester_v2"
```
Comment (dnfield) [inline shell/testing/BUILD.gn:42]: Another nit: update the output_name too.

Comment (jonahwilliams) [reply shell/testing/BUILD.gn:42]: lutter_tester_testing_factional_translation?

Comment (jonahwilliams) [reply shell/testing/BUILD.gn]: done

**Comment by jonahwilliams** [comment]
I'm going to count running the existing tests with the new tester variant as tested

**Pull request merged**